utils/helpers.ts

```diff
@@ -1,3 +1,17 @@
 export function deepClone(obj) {
     return JSON.parse(JSON.stringify(obj));
-}
\ No newline at end of file
+}
+
+export function getClearingHouseFromUrl(url: string): string | null {
+    if (url.includes("abs")) {
+      return "abs";
+    } else if (url.includes("bch")) {
+      return "bch";
+    } else if (url.includes("chm")) {
+      return "chm";
+    } else if (url.includes("ort")) {
+      return "ort";
+    }
```

Comment:
I guess it would be better to use start with `/abs` `/bch` ...

Reply:
prepended with slash